Tema-3/script6.js

```diff
@@ -0,0 +1,9 @@
+console.log("Exercitiul 6");
+
+let mediaProfit=0;
+for( let i=0; i< pretCumparareList.length; i++) {
+   mediaProfit=pretVanzare[i]-pretCumparareList[i];
```

Comment:
Ca sa poti face media profitului ai nevoie de sumaProfitului, pe care sa o imparti la numarul de masini. Astfel vrei sa faci un sumProfit, la fiecare pas, sa calculezi si sa adaugi profitul pe fiecare masina in acea suma.

La final faci impartirea, asa cum este mai jos.

Practic totul este corect cu exceptia ca aici tu doar atribui valoarea la mediaPRofit
```js
mediaProfit=pretVanzare[i]-pretCumparareList[i];
```
iar varianta corecta ar fi cea in care ai insuma profiturile
```js
mediaProfit += pretVanzare[i]-pretCumparareList[i];
```